src/toolboxes/paintbrush/Canvas.tsx

```diff
@@ -34,56 +35,84 @@ export class PaintbrushCanvas extends Component<Props> {
     this.isPressing = false;
     this.isDrawing = false;
     this.points = [];
+
+    this.state = { cursor: "none", hideBackCanvas: false };
   }
 
+
   handlePointerMove = (canvasX: number, canvasY: number): void => {
     const { x, y } = canvasToImage(
       canvasX,
       canvasY,
       this.props.imageWidth,
       this.props.imageHeight,
       this.props.scaleAndPan,
-      this.props.canvasPositionAndSize
+      this.props.canvasPositionAndSize,
     );
 
     if (this.isPressing && !this.isDrawing) {
       // Start drawing and add point
       this.isDrawing = true;
-      this.points.push({x, y});
+      this.points.push({ x, y });
     }
 
-    if (this.isDrawing) {
-      // Add new point
-      this.points.push({x, y});
+
+    if (this.props.brushType == "eraser") {
+      if (this.isDrawing) {
+        // Add new point
+        this.points.push({ x, y });
+
+        // Draw current points
+        this.drawPoints(
+          this.points,
+          "#0000FF",
```

Comment:
What is this hardwired colour for? Does this mean the eraser colour is bright green?

Reply:
No, you don't see the eraser as it's a subtraction anyway